components/apimgt/org.wso2.carbon.apimgt.rest.api.store/src/main/java/org/wso2/carbon/apimgt/rest/api/store/impl/SdkGenLanguagesApiServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package org.wso2.carbon.apimgt.rest.api.store.impl;
+
+import org.wso2.carbon.apimgt.core.configuration.APIMConfigurationService;
+import org.wso2.carbon.apimgt.core.configuration.models.SdkLanguageConfigurations;
+import org.wso2.carbon.apimgt.rest.api.store.SdkGenLanguagesApiService;
+
+
+import org.wso2.carbon.apimgt.rest.api.store.NotFoundException;
+
+import java.util.Set;
+
+import org.wso2.msf4j.Request;
+import javax.ws.rs.core.Response;
+
+public class SdkGenLanguagesApiServiceImpl extends SdkGenLanguagesApiService {
+    @Override
+    public Response sdkGenLanguagesGet(Request request) throws NotFoundException {
+        SdkLanguageConfigurations sdkLanguageConfigurations = APIMConfigurationService
```

Comment:
IMO, this is wrong. You are directly calling the configurations here. Please introduce a method in ApiStoreSdkGenerationManager and the get required values.